homeassistant/components/brother/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "domain": "brother",
+  "name": "Brother Printer",
+  "documentation": "https://www.home-assistant.io/integrations/brother",
+  "dependencies": [],
+  "codeowners": ["@bieniu"],
+  "requirements": ["pysnmp==4.4.12", "brother==0.1.4"],
```

Comment:
`pysnmp` is a dependency of the `brother` package and should not be explicitly added here as it's not used directly.

Reply:
@andrewsayre Fixed